numpy/core/code_generators/numpy_api.py

```diff
@@ -370,6 +370,8 @@ def get_annotations():
     'PyDataMem_SetHandler':                 (304, MinVersion("1.22")),
     'PyDataMem_GetHandler':                 (305, MinVersion("1.22")),
     # End 1.22 API
+    'PyArray_MatrixTranspose':              (306, MinVersion("1.25")),  # TODO: What does `MinVersion` do?
```

Comment:
I've done some reading on what `MinVersion` does. It's defined in numpy/core/code_generators/genapi.py

Reply:
That is correct, but I am not sure I feel we need to add this?  At some point people aimed to have a full featured C-API, I at least don't really feel there is a point for many convenience functions in C.

Reply:
MinVersion says that this is the first version that defines the function.

Reply:
> but I am not sure I feel we need to add this? 

I've decided it's not worth adding. My idea was to copy `np.transpose` so that I have something to go off. That isn't the way the reviewers want me to go, I'll leave it out.

> MinVersion says that this is the first version that defines the function.

Thanks